apps/condo/domains/common/hooks/useGlobalHints.tsx

```diff
@@ -0,0 +1,99 @@
+import { Col, Row, RowProps } from 'antd'
+import get from 'lodash/get'
+import isBoolean from 'lodash/isBoolean'
+import isString from 'lodash/isString'
+import getConfig from 'next/config'
+import { useRouter } from 'next/router'
+import { CSSProperties, useCallback, useEffect, useMemo, useState } from 'react'
+
+import { useAuth } from '@open-condo/next/auth'
+import { useIntl } from '@open-condo/next/intl'
+import { Banner, Carousel, Typography } from '@open-condo/ui'
+import { colors } from '@open-condo/ui/dist/colors'
+
+
+interface Hint {
+    title: string
+    description: string
+    image: string
+    color: string
+    targetUrl: string
+    invertedTextColor: boolean
+}
+
+const { publicRuntimeConfig: { globalHints } } = getConfig()
+
+const HINTS_WRAPPER_STYLE: CSSProperties = { marginBottom: 40 }
+const HINTS_ROW_GUTTERS: RowProps['gutter'] = [0, 8]
+
+export const useGlobalHints = () => {
+    const intl = useIntl()
+    const CanDisableHintsMessage = intl.formatMessage({ id: 'global.globalHints.canDisableHints' })
+    const InProfileMessage = intl.formatMessage({ id: 'global.globalHints.canDisableHints.inProfile' })
+    const MoreMessage = intl.formatMessage({ id: 'global.globalHints.banner.more' })
+
+    const [hints, setHints] = useState<null | Hint[]>(null)
+
+    const { user } = useAuth()
+    const { route } = useRouter()
+
+    const locale = intl.locale
+    const showHints = get(user, 'showGlobalHints', false)
+
+    const handleBannerClick = useCallback((url) => () => {
+        if (!url || !isString(url)) return
+        
+        window.open(url)
+    }, [])
+
+    useEffect(() => {
+        if (showHints) {
+            const pages = get(globalHints, 'pages', []) || []
+            setHints(get(pages.find(page => get(page, 'routeTemplate') === route), 'hints', null) || null)
```

Comment:
Why equality check instead of regex? What if I wanna use it on /id page or so?

Reply:
These are templates as in Next

Example: `/page`, `/page/[id]`, `/page/[id]/update`

Reply:
Got it, thanks for clarification 😄